src/Stack/Docker.hs

```diff
@@ -255,16 +255,16 @@ runContainerAndExit = do
          [["create"
           ,"--net=host"
           ,"-e",inContainerEnvVar ++ "=1"
-          ,"-e",stackRootEnvVar ++ "=" ++ toFilePathNoTrailingSep stackRoot
+          ,"-e",stackRootEnvVar ++ "=" ++ toLinuxStylePath (toFilePathNoTrailingSep stackRoot)
           ,"-e",platformVariantEnvVar ++ "=dk" ++ platformVariant
-          ,"-e","HOME=" ++ toFilePathNoTrailingSep sandboxHomeDir
+          ,"-e","HOME=" ++ toLinuxStylePath (toFilePathNoTrailingSep sandboxHomeDir)
           ,"-e","PATH=" ++ T.unpack newPathEnv
```

Comment:
I will check why this is unchanged.

Reply:
Problems with PATH turned out to be more complicated  - I add the fix to newPathEnv creation.